examples/dynamic_watcher.rs

```diff
@@ -19,19 +19,11 @@ async fn main() -> anyhow::Result<()> {
 
     // Turn them into a GVK
     let gvk = GroupVersionKind::gvk(&group, &version, &kind);
-    let mut api_resource = ApiResource::from_gvk(&gvk);
+    // Use API discovery to identify more information about the type (like its plural)
+    let (ar, _caps) = discovery::Oneshot::gvk(&client, &gvk).await?;
```

Comment:
Note: Stopped recommending people construct `ApiResource`s manually when using `DynamicObject`s and recommended doing the basic oneshot discovery to find the plural.